noxfile.py

```diff
@@ -418,7 +419,7 @@ def showcase_alternative_templates(session):
         session,
         templates=templates,
         other_opts=("old-naming",),
-        env={"GAPIC_PYTHON_ASYNC": "False", "IGNORE_FILE": "test_universe_domain.py"},
+        env={"GAPIC_PYTHON_ASYNC": "False", "IGNORE_FILES": "test_universe_domain.py,test_request_metadata.py"},
```

Comment:
This will be removed once the design is approved. I excluded ads templates during the prototype step.